packages/atomic/src/components/insight/atomic-insight-full-search-button/atomic-insight-full-search-button.pcss

```
@@ -0,0 +1,20 @@
+@import '../../../global/global.pcss';
+
+:host {
+  --full-search-background-color: #949aa2;
```

The contrast ratio of this color against a perfectly white background is really bad, I think we should use a darker grey here.

Also, if possible, I think we should avoid using locally-defined colors and re-use variables from `coveo.css`/`accessibility.css`. I think it's fine to re-defined variables though.

Example:
```suggestion
  --full-search-background-color: var(--atomic-neutral-dark);
```

---

The colours need to match the button in salesforce so I used their colours: https://github.com/coveo/salesforceintegration/tree/develop/force-app/main/default/lwc/openFullSearchButton

---

I changed the colours but we want it to darken wen you mouse over so i added a darker colour, let me know what you think